omics/analyzer/omics-run-analyzer

```diff
@@ -0,0 +1,460 @@
+#!/usr/bin/env python3
+"""
+Generate statistics for a completed HealthOmics workflow run
+
+Usage: omics-run-analyzer [<runId>...]
+                          [--profile=<profile>]
+                          [--region=<region>]
+                          [--time=<interval>]
+                          [--show]
+                          [--timeline]
+                          [--file=<path>]
+                          [--out=<path>]
+                          [--help]
+
+Options:
+ -p, --profile=<profile>  AWS profile
+ -r, --region=<region>    AWS region
+ -t, --time=<interval>    Select runs over a time interval [default: 1day]
+ -s, --show               Show run resources with no post-processing (JSON)
+ -T, --timeline           Show workflow run timeline
+ -f, --file=<path>        Load input from file
+ -o, --out=<path>         Write output to file
+ -h, --help               Show help text
+
+Examples:
+ # Show workflow runs that were running in the last 5 days
+ omics-run-analyzer -t5d
+ # Retrieve and analyze a specific workflow run by ID
+ omics-run-analyzer 1234567 -o run-1234567.csv
+ # Retrieve and analyze a specific workflow run by ID and UUID
+ omics-run-analyzer 2345678:12345678-1234-5678-9012-123456789012
+ # Output workflow run and tasks in JSON format
+ omics-run-analyzer 1234567 -s -o run-1234567.json
+"""
+import csv
+import datetime
+import json
+import os
+import re
+import sys
+
+import boto3
+import dateutil
+import docopt
```

Comment:
Why use `docopt` instead of `argparse` in the Stdlib?

Reply:
This would be blocking since `docopt` would need to be added as a dependency to the package.

Reply:
Ah, good catch. I'll add those dependencies. `docopt` is much more powerful and flexible than `argparse`